### DISCUSSION TITLE
Unexpected behavior with source checkpoints and prompts

### DISCUSSION THREAD

**Discussion by mostlyhuman** [d8ahazard/sd_dreambooth_extension#632] [Q&A]
I created a model for a new subject (A) and used the 2.0 768 checkpoint as the (Base), afterwards, I loaded checkpoint (A) as the main checkpoint even though it was half the size of the (Base), it would still properly display prompts from the (Base) as well as (A) which was the expected behavior.

First question, are they linked in the background but still separate? Seeing as how the filesize of (A) was half of (Base) it almost seemed like there was an invisible link between the two checkpoints yet each model was self-contained.

Then, I created a model for a new subject (B) and used (A) as the base, afterwards none of the prompts were working anymore when referencing subject (A), but prompts for the original Base and for the new (B) did work.

I then created a model for a new subject (C) using (B) as the base and then again all Base prompts still worked as well as (C) related prompts, but prompts for (A) and (B) no longer seemed to work.

Does anyone understand the relationship of what is happening to explain the behavior?

Also, trying to merge any combination of the above checkpoints results in checkpoints that load with an error, not sure what is up on this one either.

Thank you for any help!

**Comment by mostlyhuman** [comment]
Any knowers out there? Very perplexed by this. :/

**Comment by agentj01** [comment]
not sure about this but i created a lora model before the new year update and now i recreate it with the same export setting, it seems i am getting a constant overfitted model compared to the model i created before the update. It seems too much of the unet weight are applied from the trainer. Still have issue with dreambooth lora training no longer working since the update. It was working fine before that update with 10GB VRAM. I tried to revert back to a earlier commit but it doesnt work too :(
